app/controllers/miq_policy_controller.rb

```diff
@@ -534,32 +350,23 @@ def set_session_data
     session[:server_options]          = @server_options
   end
 
-  def features
-    [
-      {
-        :name     => :policy,
-        :title    => _("Policies"),
-        :role     => "policy",
-        :role_any => true
-      },
-    ].map { |hsh| ApplicationController::Feature.new_with_hash(hsh) }
-  end
-
   def breadcrumbs_options
     {
       :breadcrumbs  => [
         {:title => _("Control")},
         menu_breadcrumb,
       ].compact,
-      :not_tree     => %w[rsop export log].include?(action_name),
-      :record_title => :description,
+      :record_info    => @policy || @record,
+      :not_tree       => %w[rsop export log].include?(action_name),
```

Comment:
Do we still need this? We're not using a tree anymore ...

Reply:
> Do we still need this? We're not using a tree anymore ...

Yes we still need this, Simulation, Log and Import/Export sub-meunu screens still live in this controller

Reply:
Then we are maybe doing this in the wrong direction and we should extract those first...

Reply:
@skateman should i split those in the same PR?